packages/patternfly-4/react-core/src/components/ChipGroup/ChipGroupToolbarItem.tsx

```diff
@@ -1,31 +1,35 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/ChipGroup/chip-group';
+import { headingLevelType } from '../../helpers/heading';
 
 export interface ChipGroupToolbarItemProps extends React.HTMLProps<HTMLUListElement> {
   /**  Category name text */  
   categoryName?: string;
   /** Content rendered inside the chip text */
-  children: React.ReactNode; 
+  children: React.ReactNode;
   /** Additional classes added to the chip item */
-  className?: string;   
+  className?: string;
+  /** Set heading level to the chip item label */
+  headingLevel?: headingLevelType;
 }
 
 export const ChipGroupToolbarItem: React.FunctionComponent<ChipGroupToolbarItemProps> = ({
   categoryName = '', 
   children = null, 
   className = '', 
+  headingLevel: HeadingLevel = 'h1',
```

Comment:
Any particular reason for `h1` as the default instead of `h4` like it used to be?

Reply:
This was one of the requirements made in #2167 by @jgiardino

Reply:
Ah, right. This is what I had stated in #2167. 

> And assuming we land with defaulting to `<h1>` for #1881, then we should be consistent and do that here as well.

Apologies for not updating that issue after #1881 closed. For that update, we ended up using the same heading level that was originally defined in the component. One concern was that changing the heading level could be considered a breaking change. But also, Accordion ended up presenting a unique case in that we will update it in the future to support a variation that does not use headings (https://github.com/patternfly/patternfly-next/issues/1913). Which means when `headingLevel` is not specified, we would use the non-heading variant.

There was also a lot of conversation around default heading levels when the Accordion was getting updated. The general assumption is that this is a prop that will very likely get overlooked, and so the default value is important to consider.  @seanforyou23 was suggesting that it's probably better to default to lower heading levels for a component that wouldn't be used as the a primary page heading, since lower heading levels are less likely to conflict with the heading outline that exists on the page.  

For this update, I suggest using what we currently have—`<h4>`—and we can open a new issue in the future if we find that this default is resulting in issues with the overall heading outline on a page.

Thanks for working on this, @boaz0!